content/docs/use-cases/gitops-for-kubeflow.md

```diff
@@ -38,6 +38,13 @@ kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/$A
 ```shell
 brew install argoproj/tap/argocd
 ```
+2. For Ubuntu
```

Comment:
Thanks for adding this info! It looks like the list formatting is a little odd on the page. This was so even before you made the change, and the new info highlights the odd formatting.

I think it should be something like this:

1.  Install Argo CD to the Kubernetes Cluster ...

1. Install the CLI for Argo CD CLI:

    * On macOS: ...
    * On Ubuntu: ...

1. Set extra permissions ...
1. Expose the Argo CD API server...
1. Login using the CLI as admin user ...

I think you'll need to play around with the indentation to make the above format work. It's often also useful to remove the  `shell` style declaration next to the three ticks that start the code blocks, as the `shell` declaration doesn't play well with the list styling.

Reply:
Sure, will do that.